hubspot3/contacts.py

```diff
@@ -251,6 +251,46 @@ def _get_recent(
 
         return output[:limit]
 
+    def get_recently_modified_in_interval(
+            self,
+            extra_properties: Union[list, str] = None,
+            start_date: int = 0,  # data pull begin time
+            end_date: int = 0,  # data pull end time
+            **options
+    ):
+        """
+        return a list of either recently created or recently modified/created contacts by timestamp
+        :see: https://developers.hubspot.com/docs/methods/contacts/get_recently_updated_contacts
+        """
+        finished = False
+        query_limit = 100  # max according to the docs
+        # default properties to fetch
+        properties = set(self.default_batch_properties)
+        # append extras if they exist
+        if extra_properties:
+            if isinstance(extra_properties, list):
+                properties.update(extra_properties)
+            if isinstance(extra_properties, str):
+                properties.add(extra_properties)
+        time_offset = end_date
+
+        while not finished:
+            params = {"count": query_limit, "property": properties, "timeOffset": time_offset}
```

Comment:
```suggestion
            params = {"count": query_limit, 
                      "property": properties, 
                      "timeOffset": time_offset}
```
Acho melhor quebrar esse dict, assim fica mais fácil de ler (com os argumentos alinhados).